README.md

```diff
@@ -68,13 +74,16 @@ Please be respectful of the people who have created these resources. If you want
     * [Data Driven Process Systems Engineering](https://github.com/DDPSE/GT_DataDrivenPSE_Course) - Georgia Tech, Prof. Fani Boukouvala.
     * [Introduction to Experimental and Computational Research in Chemical Engineering](https://journals.flvc.org/cee/article/view/115593)
     * Machine Learning for Molecular Engineering (MIT) - [Spring 2021](https://github.com/wwang2/ML4MolEng), [Spring 2022](https://github.com/vikram-sundar/ML4MolEng_Spring2022), [Spring 2023](https://github.com/YitongTseo/ML4MolEng_Spring2023)
+    * [DWSIM - Process Eng](https://github.com/virajdesai0309/DWSim-Repo) - Process Eng. calculation like control valve sizing , hx design etc.
```

Comment:
same as above, I think these belong in a different section. I intended for this section to contain sites that contain most/all of the material for a course, rather than just single tools relevant to a course